src/services/search.js

```diff
@@ -64,6 +64,15 @@ class Search {
   }
 }
 
-const search = new Search()
+const stubSearch = {
```

Comment:
I will try to do a PR dedicated to improve first-time contributors experience. I'll use these lines in it